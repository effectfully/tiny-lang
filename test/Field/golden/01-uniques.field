let a = a;
let a = a;
for a = 1 to 2 do
    let a = a;
    let a = a;
end;
let a = a;
let a = a;
