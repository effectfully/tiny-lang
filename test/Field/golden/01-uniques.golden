let a_1 = a_0;
let a_2 = a_1;
for a_3 = 1 to 2 do
let a_4 = a_3;
let a_5 = a_4;
end;
let a_6 = a_5;
let a_7 = a_6;
